src/js/common/components/RequiredLoginView.jsx

```diff
@@ -32,15 +30,15 @@ class RequiredLoginView extends React.Component {
       return <LoadingIndicator setFocus message="Loading your information..."/>;
     }
 
-    const loginComponent = <LoginPrompt loginUrl={this.props.loginUrl}/>;
-    const verifyComponent = <VerifyPrompt verifyUrl={this.props.verifyUrl}/>;
+    const currentPath = window.location.pathname;
```

Comment:
Could leverage `appendQuery` here. Since the point is to get the path for the next query, could be something like:

```
const nextQuery = { next: window.location.pathname };
const signInUrl = appendQuery('signin', nextQuery);
const verifyUrl = appendQuery('verify', nextQuery);
```